### PR TITLE
fix(path) : fix PHP paths in backup script

### PR DESCRIPTION
**Pull request by miteto**
MON-2917

- Fix PHP CLI executable path
- Fix php.ini path

### CODE REVIEW

File: cron/centreon-backup.pl [hunk 1]
```diff
@@ -312,12 +312,12 @@ ()
 	my @tab_php_ini;
 
 	# PHP CLI
-	my $result = `php -r 'echo php_ini_loaded_file();'`;
+	my $result = `/opt/rh/rh-php71/root/usr/bin/php -r 'echo php_ini_loaded_file();'`;
```
Comment (ganoze) [inline cron/centreon-backup.pl]: Please use the @PHP_BIN@ macro that will be replaced automatically during packaging.

File: cron/centreon-backup.pl [hunk 2]
```diff
@@ -312,12 +312,12 @@ ()
...
 	push(@tab_php_ini, trim($result));
 
 	# Apache
-	if ( -e '/etc/php.ini') {
-		push(@tab_php_ini, '/etc/php.ini');
+	if ( -e '/etc/opt/rh/rh-php71/php.ini') {
```
Comment (ganoze) [inline cron/centreon-backup.pl]: What is the purpose of pushing this file ? It seems to be the default file loaded by the PHP binary above.

File: cron/centreon-backup.pl (hunk 1 above)
Comment (miteto) [reply cron/centreon-backup.pl]: Fixed in next commit

File: cron/centreon-backup.pl (hunk 2 above)
Comment (ganoze) [reply cron/centreon-backup.pl]: @kduret Do you agree that the additional backup of php.ini is useless ?

File: cron/centreon-backup.pl [hunk 3]
```diff
@@ -585,6 +578,11 @@ ()
 			print STDERR "Unable to copy PHP configuration file\n";
 		}
 	}
+	# PHP configuration ini files
+	`cp -pr /etc/opt/rh/rh-php71/php.d/ $TEMP_CENTRAL_ETC_DIR/php/`;
```
Comment (ganoze) [inline cron/centreon-backup.pl]: I'm not a big fan of hardcoded paths. People that are not using rh-php71 will have issues using Centreon Backup. For me this is a no go.

Is there any other way to backup the php.d directory ?

Comment (lpinsivy) [reply cron/centreon-backup.pl]: If php binary is know by system:

`PHP=$(php --info | grep "Additional .ini files parsed" | cut -d '>' -f2 | xargs dirname); echo $PHP`

If $PHP is empty, maybe make a list of test to perform (rh-php71, php-7.x-remi, ...) or try if locate exists to run PHP$(locate php.ini | egrep "php.ini$" | xargs dirname)

**Pull request merged**